src/plugins/hooks.js

```diff
@@ -87,9 +87,9 @@ module.exports = function (Plugins) {
 	Plugins.fireHook = async function (hook, params) {
 		const hookList = Plugins.loadedHooks[hook];
 		const hookType = hook.split(':')[0];
-		if (hook !== 'action:plugins.firehook') {
-			winston.verbose('[plugins/fireHook] ' + hook);
-		}
+		// if (hook !== 'action:plugins.firehook') {
```

Comment:
Maybe guard this with global.env = debug?

Reply:
added global.env==='development' check